swap/swap.go

```diff
@@ -473,11 +481,9 @@ func getIOmetrics(swap *Swap) error {
 	oldSwapOut := swap.ioHistory.swapOut
 	oldTimestamp := swap.ioHistory.timestamp
 	duration := time.Since(oldTimestamp).Seconds()
```

Comment:
On the **first** collection all of the per_sec calculations will have a divisor that is ~= the seconds since the empty time value.  Do the 'per sec' metrics need time to warm up?